### ISSUE TITLE
The route URL separator character '/' cannot appear consecutively. It must be separated by either a parameter or a literal value.

### ISSUE THREAD

**Issue by FootswordTechnologies** [Shazwazza/Articulate#50]
I am aware that there is a previous issue about this problem but I am very stuck. I am using Umbraco 7.2 with Articulate 1.05. All up to date and everything. This problem happened when I decided to try hanging a blog directly under my home page. I understand from the Wiki that this is now possible. I am now unable to get into the back office (or look at the front of my website) to remove the offending second blog. 
Sadly, I have not backed up my database for two weeks, so I am going to lose a lot of work. I have tried removing the new nodes from the database without much success. This is not an ideal way to work in any case. My database is now restored to how it was just after adding the second blog. The first blog is just under the root, as added by the Articulate package. 
I am using Visual Studio 2013.
I would be grateful for any help getting myself out of this mess!

**Comment by Shazwazza** [comment]
I'm going to close this issue, if you can replicate this issue again with steps using the latest Articulate version please let me know.

**Comment by Shazwazza** [comment]
This appears to occur when the umbraco.config file cache is not in sync. That is the only way I've seen this error produced. We'll keep an eye on if other people report this and if so, might have to add some error checking.

**Comment by Shazwazza** [comment]
@Weixight ok but any more info, steps to replicate, etc... ? I can't really help much without knowing how to replicate the problem.

**Comment by scottbabcock** [comment]
I also experienced this issue. Found out that it was due to the fact that I had not specified a search URL name. 

When Articulate called MapSearchRoute on app start up it ended up putting two slashes next to each other when mapping the search route since I didn't have anything defined. This caused the exception.

I was able to solve this by:
1. Removing the articulate.dll from the bin
2. Starting my Umbraco site (local debugging)
3. Defining the searchUrlName ("search" in my case)
4. Stopping my Umbraco site
5. Including articulate
6. Restarting my Umbraco site

Doing that made the exception go away.

I am using Articulate 3.0.2 from NuGet. Hope this info helps others or helps to solve the issue.
